src/dev/hsa/hsa_packet.hh

```diff
@@ -100,6 +100,13 @@ struct _hsa_barrier_or_packet_t
     uint64_t completion_signal;
 };
 
+struct _hsa_generic_vendor_pkt
+{
+    uint32_t padding[14];
+    uint64_t completion_signal;
+};
+static_assert(sizeof(_hsa_generic_vendor_pkt) == 64);
```

Comment:
Do we need to document why 64?

Reply:
I can change to Addr which should make it implicitly clear why.